sdk/keyvault/azure-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/DecryptResult.java

```diff
@@ -0,0 +1,23 @@
+package com.azure.security.keyvault.keys.cryptography;
+
+/**
+ * Represents the details of decrypt operation result.
+ */
+public class DecryptResult {
+    public DecryptResult(byte[] plainText) {
+        this.plainText = plainText;
+    }
+
+    /**
+     * The decrypted content.
+     */
+    private byte[] plainText;
+
+    /**
+     * Get the encrypted content.
+     * @return The decrypted content.
+     */
+    public byte[] plainText() {
+        return plainText;
```

Comment:
Should this return be cloned?

Reply:
This might need to be consistent across languages, will ask the KV team for P3 and see what they say.